images/structure.md

```diff
@@ -43,3 +43,11 @@ USER coder
 Your users can then create descendant images. These contain all of the original
 tooling and configuration installed onto the base image and the new
 customizations added by the users.
+
+## Coder assets
+
+Coder inserts assets into each workspace to enable code-server, JetBrains IDEs,
```

Comment:
```suggestion
Coder injects static assets into each workspace, including code-server, JetBrains IDEs,
```

Reply:
I think we were moving away from using the word "inject"? I'll let @khorne3 confirm.

Reply:
If that's the case, then I suppose we should make similar changes to our in-product copy too